### PR TITLE
Move status.dc to spec.dc

### PR DESCRIPTION
**Pull request by mrIncompetent**

### CODE REVIEW

File: api/pkg/crd/kubermatic/v1/cluster.go [hunk 1]
```diff
@@ -107,6 +107,7 @@ type ClusterSpec struct {
 	HumanReadableName string `json:"humanReadableName"` // HumanReadableName is the cluster name provided by the user
 	MasterVersion     string `json:"masterVersion"`
 	WorkerName        string `json:"workerName"` // WorkerName is a cluster used in development, compare --worker-name flag.
+	SeedDC            string `json:"seedDC"`
```
Comment (lbb) [inline api/pkg/crd/kubermatic/v1/cluster.go]: Can this be `seedDatacenterName` or `seedDatacenter`

**Review by lbb — CHANGES_REQUESTED** [review]

File: api/pkg/crd/kubermatic/v1/cluster.go (hunk 1 above)
Comment (mrIncompetent) [reply api/pkg/crd/kubermatic/v1/cluster.go]: Fixed

**Review by lbb — APPROVED** [review]
👍

**Review by metalmatze — APPROVED** [review]
LGTM

**Pull request merged**